src/components/checkbox/Checkbox.tsx

```diff
@@ -122,10 +124,10 @@ export const Checkbox = ({
       checked={checked}
       className={['amino-input-wrapper', disabled ? 'disabled' : ''].join(' ')}
       data-testid={testId}
-      htmlFor={label}
+      htmlFor={labelAsHtmlAttribute}
```

Comment:
Maybe you can just use `testId` variable instead 🤔

Reply:
I think if we are using labels in forms, that id is more semantically correct. It would be more functionally correct as well, but we wrap the switch/checkbox in the whole label anyway so it doesn't make a difference.

Reply:
Sounds good 😉